### PR TITLE
Build layouts and homepage for user

### PR DESCRIPTION
**Pull request by ovpdng124**
Use template on w3layouts.com for this project

### CODE REVIEW

**Comment by ovpdng124** [comment]
ready

File: resources/views/layouts/users/parts/header.blade.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    </button>
+                </div>
+            </div>
+            <!-- //cart details -->
+            <div class="clearfix"></div>
+        </div>
+        <div class="clearfix"></div>
+    </div>
+</div>
+<div class="ban-top" style="height: 4em">
```
Comment (quoctd-0607) [inline resources/views/layouts/users/parts/header.blade.php]: style viết ra file css đi em

File: resources/views/users/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
+@extends('layouts.users.master_layout')
+
+@section('title')
+    {{trans('messages.homepage')}}
+@endsection
+
+@section('content')
+    <!-- banner -->
+    <div style="height: 500px" class="container-fluid">
```
Comment (quoctd-0607) [inline resources/views/users/index.blade.php]: tương tự

File: resources/views/users/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <!-- top Products -->
+    <div class="ads-grid">
+        <div class="container">
+            <!-- tittle heading -->
+            <h3 class="tittle-w3l">{{trans('messages.top_product')}}
+                <span class="heading-style">
+					<i></i>
+					<i></i>
+					<i></i>
+				</span>
```
Comment (quoctd-0607) [inline resources/views/users/index.blade.php]: indent. fix hết indent nhé e

**Comment by ovpdng124** [comment]
ready

**Pull request merged**